models/DataObject/ClassDefinition/Data/ManyToManyObjectRelation.php

```diff
@@ -75,6 +75,11 @@ class ManyToManyObjectRelation extends AbstractRelations implements QueryResourc
      */
     public $visibleFields;
 
+    /**
+     * @var bool
+     */
+    public $allowToCreateNewObject = false;
```

Comment:
Default should be true as it is the current behavior, otherwise this would be a breaking change.